internal/uniqid/uniqid.go

```diff
@@ -0,0 +1,202 @@
+package uniqid
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/google/uuid"
+)
+
+// DirLocation represents tokens to indicate where uniqid file should be
+// located.
+type DirLocation int
+
+// DirLocation enums.
+const (
+	InHome DirLocation = iota
+	InTmp
+)
+
+const (
+	fileName = "uniqid"
+	asDir    = "activestate_dat"
```

Comment:
```suggestion
	asDir    = "activestate/persist"
```

Reply:
I think this is a bit friendlier to the user. Still a single main dir, and the subdir somewhat communicates its intend.